docs/operating.adoc

```diff
@@ -1,7 +1,32 @@
+:start-script: https://github.com/kroxylicious/kroxylicious/blob/main/kroxylicious-app/src/assembly/kroxylicious-start.sh
 = Operating proxies
 
 == Logging
 
+The Kroxylicious https://github.com/kroxylicious/kroxylicious/releases/latest[binary distributions] ship with https://logging.apache.org/log4j/2.x[log4j2] as a logging backend. To customize the logging configuration:
+
+=== 1. Use an Alternative Log4j configuration file
+
+When using {start-script}[bin/kroxylicious-start.sh] from the binary distribution, you can optionally set an environment variable:
+
+[source,shell]
+----
+KROXYLICIOUS_LOGGING_OPTIONS="-Dlog4j2.configurationFile=/path/to/custom/log4j2.yaml"
```

Comment:
If its really the logging options variable wouldn't we just expect the `/path/to/custom/log4j2.yaml`? Given its just specifying a property for the JVM maybe it should just be `KROXYLICIOUS_JVM_OPTIONS` or `KROXYLICIOUS_ADDITIONAL_JVM_OPTIONS` ?

Reply:
I'd support refactoring to `KROXYLICIOUS_LOG_CONFIG_PATH`. The current entry script does:

```
if [ "${KROXYLICIOUS_LOGGING_OPTIONS+set}" != set ]; then
  KROXYLICIOUS_LOGGING_OPTIONS="-Dlog4j2.configurationFile=$(script_dir)/../config/log4j2.yaml"
fi
export JAVA_OPTIONS="${KROXYLICIOUS_LOGGING_OPTIONS} ${JAVA_OPTIONS:-}"
```

`JAVA_OPTIONS` is currently how the user could set additional JVM config by env var. Having a `KROXYLICIOUS_ADDITIONAL_JVM_OPTIONS` sounds like a better idea, though I think we should keep them separate so that the user doesn't blow away logging configuration by configuring `Xmx`.

Reply:
Does it that simplify things to 
```shell
export JAVA_OPTIONS="-Dlog4j2.configurationFile=${KROXYLICIOUS_LOG_CONFIG_PATH:$(script_dir)/../config/log4j2.yaml} ${JAVA_OPTIONS:-}"
```

For various definitions of simplify...